workflow-dev/tpl/generate_params.toml

```diff
@@ -17,37 +17,37 @@
 # - azure: Store persistent data in Azure's object storage
 # - gcs: Store persistent data in Google Cloud Storage
 # - minio: Store persistent data on in-cluster Minio server
-storage = "minio"
+storage = "{{ env "STORAGE_TYPE" | default "minio"}}"
 
 [minio]
 org = "deisci"
 pullPolicy = "Always"
 dockerTag = "canary"
 
 [s3]
-accesskey = "YOUR KEY HERE"
-secretkey = "YOUR SECRET HERE"
+accesskey = "{{ env "AWS_ACCESS_KEY" }}"
+secretkey = "{{ env "AWS_SECRET_KEY" }}"
 # Any S3 region
-region = "us-west-1"
+region = "{{ env "REGION" | default "us-west-1" }}"
 # Your buckets.
-registry_bucket = "your-registry-bucket-name"
-database_bucket = "your-database-bucket-name"
-builder_bucket = "your-builder-bucket-name"
+registry_bucket = "{{ env "REGISTRY_BUCKET" | randAlpha 5 }}"
+database_bucket = "{{ env "DATABASE_BUCKET" | randAlpha 5 }}"
+builder_bucket = "{{ env "BUILDER_BUCKET" | randAlpha 5 }}"
 
 [azure]
-accountname = "YOUR ACCOUNT NAME"
-accountkey = "YOUR ACCOUNT KEY"
-registry_container = "your-registry-container-name"
-database_container = "your-database-container-name"
-builder_container = "your-builder-container-name"
+accountname = "{{ env "ACCOUNT_NAME" }}"
+accountkey = "{{ env "ACCOUNT_KEY" }}"
+registry_container = "{{ env "REGISTRY_BUCKET" | randAlpha 10 }}"
+database_container = "{{ env "DATABASE_BUCKET" | randAlpha 10 }}"
+builder_container = "{{ env "BUILDER_BUCKET" | randAlpha 10 }}"
```

Comment:
since you prefixed the s3 env vars with `AWS_`, maybe it'd be a good idea to prefix these with `AZURE_` and the below with `GCS_`. Thoughts?